awsx/schema.json

```diff
@@ -1689,6 +1689,24 @@
                     },
                     "description": "The Route Table Associations for the VPC."
                 },
+                "publicRouteTableIds": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "privateRouteTableIds": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "isolatedRouteTableIds": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
```

Comment:
The schema.json is generated by the `schemagen` program. The VPC types are defined using this code here: https://github.com/pulumi/pulumi-awsx/blob/master/schemagen/pkg/gen/ec2.go